src/rules/no-empty-source/README.md

```diff
@@ -0,0 +1,35 @@
+# no-empty-source
+
+Disallow empty source.
+
+```css
+  ···\n\t
+/**     ↑
+ *  This whitespace */
+```
+
+Sources containing only spaces or/and empty lines is considered empty.
```

Comment:
Last one correction, I promise :)

"A source containing only spaces or/and empty lines is considered empty"